src/components/Table/table-cells/TableHeadCell.tsx

```diff
@@ -16,20 +17,19 @@ export const TableHeadCell: React.FC<TableCellProps> = ({
     children,
     ...props
 }) => {
-
-    // what's the correct naming convention for these cell scss classes?
     return (
         <Typography
+            {...props}
             className={clsx(
-                'moonstone-TableCell-border',
+                {'flexFluid': typeof width === 'undefined'},
                 'textAlign' + capitalize(textAlign),
                 'moonstone-verticalAlign' + capitalize(verticalAlign),
                 className
             )}
             component={component}
             weight="bold"
             variant="body"
-            {...props}
+            style={{width, ...props.style}}
```

Comment:
might need to reverse this if width needs to override what's in props.style

Reply:
Nice catch. Yes, you're right I reversed the order